development-tools/clinic.rst

```diff
@@ -2000,52 +2000,73 @@ The generated glue code looks like this:
 .. versionadded:: 3.13
 
 
-.. _clinic-howto-getter:
+.. _clinic-howto-pygetsetdef:
 
-How to generate a getter
-------------------------
+How to declare PyGetSetDef functions
+------------------------------------
 
-"Getters" are C functions that facilitate property-like access for a class.
-See :c:type:`getter <PyGetSetDef>` for details.
-You can use the ``@getter`` directive to generate an "impl" function for a
-getter using Argument Clinic.
+"Getters" and "setters" are C functions that facilitate property-like access for a class.
+See :c:type:`getter <PyGetSetDef>` and :c:type:`setter <PyGetSetDef>` for details.
```

Comment:
```suggestion
"Getters" and "setters" are C functions defined in a :c:type:`PyGetSetDef` struct
 that facilitate property-like access for a class.
```
I'm not 100% sure about my proposed wording, but currently I saw two (minor) issues here:
* both "getter" and "setter" link to the same thing (and it's not particularly evident what the thing is);
* the paragraph sounds a bit "choppy", with 3 short sentences that repeat "getter" and "setter".

Reply:
Yeah your suggestion looks better. Not using repeatable word is quite difficult to me :)